config/environments/development.rb

```diff
@@ -17,21 +17,10 @@
   # Enable server timing
   config.server_timing = true
 
-  # Enable/disable caching. By default caching is disabled.
-  # Run rails dev:cache to toggle caching.
-  if Rails.root.join("tmp/caching-dev.txt").exist?
-    config.action_controller.perform_caching = true
-    config.action_controller.enable_fragment_cache_logging = true
-
-    config.cache_store = :memory_store
-    config.public_file_server.headers = {
-      "Cache-Control" => "public, max-age=#{2.days.to_i}",
-    }
-  else
-    config.action_controller.perform_caching = false
-
-    config.cache_store = :null_store
-  end
+  # config.session_store :cache_store, key: '_app_session_key'
+  config.session_store :redis_session_store, 
+    servers: 'redis://localhost:6379/1',
```

Comment:
👀 

I think this should be `ENV['REDIS_URL']`? though this is only in development rather than the application config

Reply:
good catch, just fixed that